src/broker/rundir.h

```diff
@@ -20,4 +23,6 @@ int rundir_create (attr_t *attrs,
                    const char *tmpdir,
                    flux_error_t *error);
 
+#endif // !BROKER_RUNDIR
+
```

Comment:
nit: comment should be `!BROKER_RUNDIR_H`.

Also, for consistency with all other files could use C style comment: `/* !BROKER_RUNDIR_H */`